app/models/mesh/Mesh.scala

```diff
@@ -89,8 +89,8 @@ class MeshDAO @Inject()(sqlClient: SQLClient)(implicit ec: ExecutionContext)
       accessQuery <- readAccessQuery
       rList <- run(
         sql"select #$infoColumns from #$existingCollectionName where _id = ${id.id} and #$accessQuery".as[InfoTuple])
-      r <- rList.headOption.toFox ?~> ("Could not find object " + id + " in " + collectionName)
-      parsed <- parseInfo(r) ?~> ("SQLDAO Error: Could not parse database row for object " + id + " in " + collectionName)
+      r <- rList.headOption.toFox
```

Comment:
Is there reason, you removed the error messages here? Since they can't use the helper functions, I feel like manual error messages would be nice here :slightly_smiling_face:

Reply:
I believe they were put in there to debug a specific thing some time ago. Since this was solved I’d rather keep the code clean. I don’t think this adds that much. But I don’t have a strong opinion, if you say I should re-add it, I will :)

Reply:
Alright, then don't re-add them.